<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LD-4V2P3G2" name="Default test" tests="1" failures="0" timestamp="14 Mar 2018 22:13:42 GMT" time="44.422" errors="0">
  <testcase name="@BeforeTest setUp" time="0.0" classname="com.rf.test.website.LSD.LSDCustomerTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.rf.core.utils.DBUtil.getConnectionString(DBUtil.java:49)
	at com.rf.core.utils.DBUtil.getDataSource(DBUtil.java:97)
	at com.rf.core.utils.DBUtil.performDatabaseQuery(DBUtil.java:77)
	at com.rf.test.website.LSD.LSDCustomerTests.setUp(LSDCustomerTests.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.beforeRun(TestRunner.java:641)
	at org.testng.TestRunner.run(TestRunner.java:609)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase name="@BeforeClass beforeClass" time="0.0" classname="com.rf.test.website.LSD.LSDCustomerTests">
    <skipped/>
  </testcase> <!-- @BeforeClass beforeClass -->
  <testcase name="@BeforeMethod initiateSoftAssertObject" time="0.0" classname="com.rf.test.website.LSD.LSDCustomerTests">
    <skipped/>
  </testcase> <!-- @BeforeMethod initiateSoftAssertObject -->
  <testcase name="@AfterMethod tearDownAfterMethod" time="0.0" classname="com.rf.test.website.LSD.LSDCustomerTests">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDownAfterMethod -->
  <testcase name="@AfterClass afterClass" time="0.0" classname="com.rf.test.website.LSD.LSDCustomerTests">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="TotalNumberOfAllPCs_3349q" time="0.256" classname="com.rf.test.website.LSD.LSDCustomerTests">
    <skipped/>
  </testcase> <!-- TotalNumberOfAllPCs_3349q -->
</testsuite> <!-- Default test -->
